qemu/tests/cfg/pvpanic.cfg

```diff
@@ -0,0 +1,38 @@
+- pvpanic: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = driver_in_use
+    run_bgstress = pvpanic
+    disable_shutdown = yes
+    monitor_type = qmp
+    event_check = "GUEST_PANICKED"
+    driver_name = "pvpanic"
+    suppress_exception = yes
+    crash_method = nmi
+    timeout = 360
+    wait_bg_time = 720
+    run_bg_flag = "before_bg_test" 
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    set_kdump_cmd = "chkconfig kdump off && systemctl stop kdump.service"
```

Comment:
Instead of 'chkconfig kdump off' it would be better to use 'systemctl disable kdump'